### ISSUE TITLE
Verbose / Debugging Output Should Show Profiling Attributes

### ISSUE THREAD

**Issue by fsundermeyer** [openSUSE/daps#289]
... if set...

**Comment by tomschr** [comment]
Please also integrate the SUSE stylesheet version, if available. Maybe something like this:

```
        DAPS VERSION: 2.1.3
  DB-STYLESHEET-VERS: 1.78.1
SUSE-STYLESHEET-VERS: 2.0.3
```

**Comment by fsundermeyer** [comment]
Fixed with commit   c40c2a2
